instruments/qubitekk/cc1.py

```diff
@@ -49,11 +49,15 @@ def __init__(self, filelike):
             self._bool = ("ON", "OFF")
             self._set_fmt = ":{}:{}"
             self.TriggerMode = self._TriggerModeNew
+            self._ack_expected = self.repeat
         else:
             self._bool = ("1", "0")
             self._set_fmt = ":{} {}"
             self.TriggerMode = self._TriggerModeOld
 
+    def repeat(self, msg=""):
```

Comment:
Just overwrite the `CC1._ack_expected` function instead of making a new one (`self.repeat`) and pointing `self._ack_expected` at it